modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceEntryLocalServiceImpl.java

```diff
@@ -613,36 +607,10 @@ public int searchCommercePriceEntriesCount(
 		return _searchCommercePriceEntriesCount(searchContext);
 	}
 
-	@Indexable(type = IndexableType.REINDEX)
 	@Override
-	public CommercePriceEntry setHasTierPrice(
-			long commercePriceEntryId, boolean hasTierPrice)
-		throws PortalException {
-
-		CommercePriceEntry commercePriceEntry =
-			commercePriceEntryPersistence.findByPrimaryKey(
-				commercePriceEntryId);
-
-		commercePriceEntry.setBulkPricing(true);
-		commercePriceEntry.setHasTierPrice(hasTierPrice);
-
-		return commercePriceEntryPersistence.update(commercePriceEntry);
-	}
-
-	@Override
-	public CommercePriceEntry setHasTierPrice(
-			long commercePriceEntryId, boolean hasTierPrice,
-			boolean bulkPricing)
-		throws PortalException {
-
-		CommercePriceEntry commercePriceEntry =
-			commercePriceEntryPersistence.findByPrimaryKey(
-				commercePriceEntryId);
-
-		commercePriceEntry.setBulkPricing(bulkPricing);
-		commercePriceEntry.setHasTierPrice(hasTierPrice);
-
-		return commercePriceEntryPersistence.update(commercePriceEntry);
+	public void updateCommercePriceEntries() throws PortalException {
```

Comment:
how many times is called this method? Just thinking to remove it and call the two methods directly